ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/QBFTGenesisFileModule.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright ConsenSys AG.
```

Comment:
New files to have updated copyright for Contributors to Hyperledger Besu

Reply:
Thank you, updated.